src/main/java/com/mraof/minestuck/world/storage/loot/OverworldTableLootEntry.java

```diff
@@ -0,0 +1,180 @@
+package com.mraof.minestuck.world.storage.loot;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSerializationContext;
+import com.mraof.minestuck.Minestuck;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.JSONUtils;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.storage.loot.*;
+import net.minecraft.world.storage.loot.conditions.ILootCondition;
+import net.minecraftforge.fml.common.ObfuscationReflectionHelper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.lang.reflect.Field;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class OverworldTableLootEntry extends LootEntry
```

Comment:
I don't think this is something you want to use. What is your use case?

Reply:
I do not fully understand how to autogenerate the loot table json and created this from the Lands loot entry class assuming it was needed. If this can be removed then I will do so but i do not know the repurcussions

Reply:
It was used in the land loot tables to include loot from other loot tables, which it picks depending on the state of the world (i.e what land types the world has). Something like that would not be needed here, and without any other reason, I think that you should remove the entry and stick to writing the loot table normally.